README.md

```diff
@@ -66,7 +66,29 @@ file.csslint.results = []; // CSSLint errors
 file.csslint.opt = {}; // The options you passed to CSSLint
 ```
 
-## Custom Reporters
+## Using reporters
+
+Several reporters come built-in to css-lint. To use one of these reporters, pass the name to `csslint.reporter`, along with an optional file path. If the file path is included, the report will be written to this location, instead of printed in the console.
+
+For a list of all reporters supported by `csslint`, see the [csslint wiki](https://github.com/CSSLint/csslint/wiki/Command-line-interface#--format).
+
+```js
+gulp.task('lint', function() {
+  gulp.files('lib/*.css')
+    .pipe(csslint())
+    .pipe(csslint.reporter('junit-xml'));
+```
+
+If you want to print the report to disk instead of output to the console, supply the a second optional file path where the file will be written.
+
+```js
+gulp.task('lint', function() {
+  gulp.files('lib/*.css')
+    .pipe(csslint())
+    .pipe(csslint.reporter('junit-xml', 'csslint-junit-report.xml'));
```

Comment:
Not that I know of (haven't searched though). I stole the idea from the grunt csslint plugin.
How would we provide the option to save it though? I could take in an option hash instead?

Having a way to save it to disk allows general purpose tool such as SonarQube to read the file along with other junit/checkstyle files for instance, so I think it's a valid feature